.github/workflows/ci-workflow.yml

```
@@ -1,18 +1,26 @@
-name: "Continuous Integration"
+name: ci-workflow
 
 on:
   pull_request:
     types: [opened, synchronize, reopened]
 
 jobs:
   quality:
-    name: "CI: Check quality"
+    name: Check quality
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
 
       - name: Install pnpm
-        run: npm install -g pnpm
+        uses: pnpm/action-setup@v2.2.4
+        with:
+          version: 8
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.17'
+          check-latest: true
```

```suggestion
      - name: Read .nvmrc
         run: echo "##[set-output name=NVMRC;]$(cat .nvmrc)"
         id: nvm

      - name: Setup Node
        uses: actions/setup-node@v3
        with:
          node-version: "${{ steps.nvm.outputs.NVMRC }}"
          check-latest: true
```

I think it would be better to automatically track the number written in the `.nvmrc` of this project.

If it is used as a [`magic number`](https://drehzr.tistory.com/1136) like this, I think the management point will continue to increase!

---

Cool suggestion

---

@manudeli  I commented only ci-workflow but same code has in release-workflow.yml ! 😃